### PR TITLE
[ADD] option to repeat current or all sentences

### PR DESCRIPTION
**Pull request by HamadaSalhab**

### CODE REVIEW

File: lib/pages/audio_page.dart [hunk 1]
```diff
@@ -174,74 +184,115 @@ class _AudioPageState extends State<AudioPage> {
...
+                      // print(
+                      //     'for all = $repeatForAll, chosen = $chosenRepeatNumber, current = $currentRepeatNumber.');
+                    },
+                    child: const Text('OK'),
+                  ),
+                ],
+              );
+            },
+          );
+        });
```
Comment (AlmetovKamil) [inline lib/pages/audio_page.dart:295]: LGTM

File: lib/pages/audio_page.dart [hunk 2]
```diff
@@ -174,74 +184,115 @@ class _AudioPageState extends State<AudioPage> {
...
+                        flex: 4,
+                        child: TextFormField(
+                          onChanged: (value) {
+                            enteredNumber = int.tryParse(value);
+                            // print('entered number = $enteredNumber');
+                          },
+                          keyboardType: TextInputType.number,
+                          cursorColor: clr.kBnbSelectedItemClr,
+                          decoration: InputDecoration(
+                            labelText: 'repetitions',
```
Comment (AlmetovKamil) [inline lib/pages/audio_page.dart:230]: Overflow of word 'repetitions'

File: lib/pages/audio_page.dart [hunk 3]
```diff
@@ -174,74 +184,115 @@ class _AudioPageState extends State<AudioPage> {
...
+                              activeColor: clr.kOrangeAccent,
+                              value: repeatForAll,
+                              onChanged: (value) {
+                                setter(() {
+                                  repeatForAll = value!;
+                                });
+                                // print(
+                                //     'entered number on cb changed = $enteredNumber');
+                              },
+                            ),
```
Comment (AlmetovKamil) [inline lib/pages/audio_page.dart:259]: Overflow of checkbox

**Comment by AlmetovKamil** [comment]
When I enter 3 times, it repeat a sentence 4 times

**Comment by AlmetovKamil** [comment]
How can we allow a user to choose the repeat mode (repeat either only current sentence or all)?

**Comment by HamadaSalhab** [comment]
well this not a bug, but rather a conceptual issue. When you want to repeat it 3 times, it repeats 3 times and then plays the sentence. It can be easily changed

**Comment by HamadaSalhab** [comment]
Long press on the repeat button

**Review by AlmetovKamil — CHANGES_REQUESTED** [review]

**Review by AlmetovKamil — APPROVED** [review]
LGTM!

**Comment by AlmetovKamil** [comment]
Everything is good now!

**Pull request merged**